addons/purchase/models/purchase_order.py

```diff
@@ -384,14 +384,13 @@ def _notify_by_email_prepare_rendering_context(self, message, msg_vals, model_de
         )
         subtitles = [render_context['record'].name]
         # don't show price on RFQ mail
-        if self.state not in ['draft', 'sent']:
+        if self.state in ['draft', 'sent']:
             if self.date_order:
```

Comment:
`date_order` is a required field so no need to keep the condition.
LGTM apart from that. But you may need to add a test too.

Reply:
I removed the condition
regarding the test, I don't think it's really worth it for this change